src/index.ts

```diff
@@ -72,6 +72,18 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  // Respond to the GitHub app installation deleted
+  app.on("installation.deleted", async (context) =>{
+
+    // ? user story: delete repo or just the secret ?
+    // ? do we have to take care of repo
+    // TODO: check if secret exists and catch errors
+        
+    var name = "peribolos-"+context.payload.installation.id;
+    
+    await k8sApi.deleteNamespacedSecret(name, namespace);
```

Comment:
```suggestion
    await k8sApi.deleteNamespacedSecret(name, k8snamespace);
```
can we make the variable as suggested, this change would be introduced on other places via: https://github.com/open-services-group/peribolos-as-a-service/pull/21